x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/split_cards/style.scss

```diff
@@ -0,0 +1,3 @@
+.mlSplitCard__border {
+  border: 1px solid $euiColorLightShade
```

Comment:
We have a [border variable](https://elastic.github.io/eui/#/theming/borders) in EUI: `border: $euiBorderThin;`

Also include `padding-top: $euiSizeXS`.  With these changes, probably makes sense to rename.  Perhaps `.mlPickFields__splitCard`?

Reply:
Changed in https://github.com/elastic/kibana/pull/129762/commits/82accec37a8e4b8790722f38f87fed1fae3deb55